test/passThrough.js

```diff
@@ -14,7 +14,7 @@ describe('stream', function () {
     it('piping into second plumber should keep piping', function (done) {
         gulp.src(fixturesGlob)
             .pipe(plumber())
-            .pipe(gutil.noop())
+            .pipe(through2.obj())
```

Comment:
This is funny because `gutil.noop` was never a stream, it was just a noop function.

Reply:
To be honest I also was surprised by this: replacing it by an empty function caused the tests to fail. Then I noticed that it's actually calling the function: this functions returns a pass-through object stream. `new PassThrough({objectMode: true})` also worked but since `through2` was available I used it to match the old behavior:

https://github.com/gulpjs/gulp-util/blob/3c18e3cce5047e591ed353a8486ca1251b7f56c9/lib/noop.js#L4
```javascript
module.exports = function () {
  return through.obj();
};
```

If I encounter it in other libraries, I'll submit a PR to document this on `gulp-util`'s README.

Reply:
Oops, I didn't realize gutil had it as a noop stream.  I guess I need to update the readme and blog post.

Reply:
@demurgos done! thanks for pointing that out.